Multi: v2 staking design

Sirmorrison opened the pull request:
Resolves #981 
Fix #1014
Fix #976 
Fix #1005
Fix #1022 
Fix #978

- [x] Fix wallet selector page display when app loads for the first time.
- [x] Fix backup modal notification display for wallet that don't have their seed phrase backed up.
- [x] Update the staking layout to the v2 layout design
- [x] Update stake shuffle and staking page to single wallet selector approach (needed to test the account selector)
- [x] Fix crash when staking page button is clicked twice
- [x] Fix ticket list refresh bug
- [x] Fix app crash when the settings button is clicked when a watch only wallet is selected.

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/27733432/181623862-e021277e-88de-4735-881d-c25d8b643280.png">

<img width="1278" alt="image" src="https://user-images.githubusercontent.com/27733432/181623955-6a1a97ed-4ef6-4b9e-8ebd-2c6404fb3322.png">

Sirmorrison committed:
add v2 staking alance layout

Sirmorrison committed:
add tickets list

Sirmorrison committed:
set backup modal deplay on selected wallet page load

Sirmorrison committed:
Add stake statistics modal
- clean up redundant files and lines of code

Sirmorrison committed:
update ticket list item layout to v2 design

Sirmorrison committed:
fix ticket list alignment bug

Sirmorrison committed:
fix app crash as a result of vsp selector

dreacot reviewed (dismissed):
1. Navigating to the staking page cause a overall lag of the app for a couple of seconds.

![Screenshot from 2022-07-26 16-19-05](https://user-images.githubusercontent.com/25265396/181045028-b6ec4ad3-387d-4c28-84b6-dba2f8d29733.png)

2. All tickets are showing at once which isn't efficient and most likely the cause of the lag. the list should be paginated.

3. clicking on the settings icon on the staking page causes a crash

4. The statistic dialog is no same as that of the desing
![Screenshot from 2022-07-26 16-24-31](https://user-images.githubusercontent.com/25265396/181046397-b2facd2c-db25-4594-93e9-e127c0f68894.png)
![Screenshot from 2022-07-26 16-25-28](https://user-images.githubusercontent.com/25265396/181046444-dc524a5e-f9f9-4225-9d9d-574850b8799b.png)

![Screenshot from 2022-07-26 16-26-54](https://user-images.githubusercontent.com/25265396/181046807-eeb42db7-773a-4cee-b362-669dfee908c7.png)
5. there should be more right margin after the settings icon, and `stake` should be in bold

![Screenshot from 2022-07-26 16-28-58](https://user-images.githubusercontent.com/25265396/181047326-a7bed639-3361-411b-a548-ebc252451408.png)
6. This section should be titled`Tickets` and the icons don't seem to be on par with the provided figma design

7. the hover on tickets doesn't have a proper padding
![Screenshot from 2022-07-26 16-30-25](https://user-images.githubusercontent.com/25265396/181047793-fe149e19-34c4-451c-8326-ddf099681f07.png)
![Screenshot from 2022-07-26 16-30-17](https://user-images.githubusercontent.com/25265396/181047798-bc0fe3d5-ee9d-442e-a544-e7e8acbf49eb.png)

8. The staking details page doesn't yet seem to be on par with the figma design

Sirmorrison committed:
update account selector to v2 single wallet selection appraoch

Sirmorrison committed:
Update staking wallet selector
Update stake shuffle to use single wallet selector approach
Fix setting button margin bug
Fix app crash when settings button is clicked due to watch only wallet selection
Fix ticket list hover alignment bug
Add title to ticket list section
general code clean up

dreacot reviewed (commented):
clicking the settings icon isn't working, is that intentional?

JustinBeBoy reviewed (changes requested):
1. After confirm automatic Ticket Purchase the switcher not turn on then turn off without notice for user the reason. 

2. Should uppercase the first character
![image](https://user-images.githubusercontent.com/19331824/181583385-5343df66-00b7-4118-9be5-5b7cbcb9c11a.png)

JustinBeBoy commented on ui/page/staking/stake_info.go:
```diff
@@ -0,0 +1,224 @@
+								)
+							}),
+							pg.dataRows(values.String(values.StrLiveTickets), pg.ticketOverview.Unmined),
+							pg.dataRows("Can Buy", pg.CalculateTotalTicketsCanBuy()),
```
Add "Can Buy" to the localize string

Sirmorrison committed:
Hide staking buttons for watch-only wallets
- Fix default app container size as it wax too small on full screen.
- Adjust pading and localize and fix string variables typo
